ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.20.yaml

```diff
@@ -761,6 +761,56 @@ tests:
       COMPUTE_NODE_TYPE: t2a-standard-4
       OCP_ARCH: arm64
     workflow: openshift-e2e-gcp-ovn
+- as: ocp-e2e-gcp-ovn-multi-x-x-to-a-a
```

Comment:
Hey @lwan-wanglin ... small nit, this should be
```ocp-e2e-gcp-ovn-multi-x-x-to-a-x```

Reply:
@tvardema what do you think? We didn't talk about infra nodes when we came up with the naming scheme. If you're good with it, then we'll need to up date the naming doc.

Reply:
hi @tvardema, any suggestions?

Reply:
hey @lwan-wanglin Trevor and I talked offline, we felt that `ocp-e2e-gcp-ovn-multi-x-x-to-a-x` was the better way to go.  Anytime we change control plane architecture we should assume that the infra arch will change as well.

Reply:
got it, will fix it